interfaces/backends/backends.go

```diff
@@ -71,7 +72,11 @@ func backends() []interfaces.SecurityBackend {
 	// profiles that keep applications operational, in forced-devmode.
 	switch apparmor_sandbox.ProbedLevel() {
 	case apparmor_sandbox.Partial, apparmor_sandbox.Full:
-		all = append(all, &apparmor.Backend{})
+		aa := &apparmor.Backend{}
+		if release.PreseedMode() {
```

Comment:
I looked at this again,  we might not want to block this PR on this, but it seems the cleanest solutions is actually to pass a SecurityBackendOptions to SecurityBackend.Initialize  and have the only one of them be PreseedMode

Reply:
Okay, I can clean that up in a followup.